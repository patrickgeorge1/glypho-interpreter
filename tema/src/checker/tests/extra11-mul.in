100000001
1000000002
