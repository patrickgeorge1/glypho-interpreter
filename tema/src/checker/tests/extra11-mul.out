100000001200000002
